### PR TITLE
Fix #15: Show start and end dates for studies

### LINKED ISSUE #15: each study should show an associated date range
Probably a good idea to do this on both the study list and study details pages.

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

File: shielddash/core/static/js/app/components/views/date-range.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+
+export default function(props) {
+    var formatOptions = { month: "long", day: "numeric", year: "numeric" }
```
Comment (spasovski) [inline shielddash/core/static/js/app/components/views/date-range.js]: nitpick but I've been doing 2 space indents throughout the project

**Comment by spasovski** [comment]
Looks good thanks! r+wc.

File: shielddash/core/static/js/app/components/views/date-range.js (hunk 1 above)
Comment (openjck) [inline shielddash/core/static/js/app/components/views/date-range.js]: Sounds good. I'll also update _.editorconfig_ to use two spaces for JavaScript.

**Comment by openjck** [comment]
Updated! 😄

**Comment by spasovski** [comment]
Awesome thanks.

**Pull request merged**